src/main/java/uk/gov/cshr/civilservant/dto/QuizDto.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.cshr.civilservant.dto;
+
+import java.time.LocalDateTime;
+import java.util.Set;
+
+import lombok.*;
+import uk.gov.cshr.civilservant.domain.Profession;
+import uk.gov.cshr.civilservant.domain.Status;
+
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class QuizDto {
+  private Long id;
+  private String name;
+  private Profession profession;
+  private Set<QuestionDto> questions;
```

Comment:
Should be list since questions are ordered

Reply:
Refer previous reply. We do not care about the order of the questions.

Reply:
What about the admin viewing the questions they created, will they be shuffled too?

Reply:
@m24murray I think what Ahmed asking is valid. Do we have a requirement to present the questions on the creation order. We could stick in a created_on timestamp in the question record and order them while presenting.

Reply:
preview for admin queries by question id, so this discussion can be closed